### DISCUSSION TITLE
Tell me letter of hiragana you like!!  あなたが好きなひらがな一文字を教えてください！！

### DISCUSSION THREAD

**Discussion by natsuonorepositories** [NAIST-SE/AttractingContributors#509] [General]
I would like to know what letter of hiragana you like.
あなたの好きなひらがな一文字が知りたいです。

If you help with my project, please add your favorite hiragana to the List of Hobby.
もし、このプロジェクトにご協力いただけるなら、あなたのお気に入りのひらがなを「List of Hobby」に追加してください。

https://github.com/natsuonorepositories/AttractingContributors-hiragana

**Comment by kazuyo000207** [comment]
I’ve added  ‘ん’ and made a pull request.
Would you mind contributing mine too https://github.com/kazuyo000207/AttractingContributors-Curry.git
